### ISSUE TITLE
Accessibility:: Create A11y Issue Failure Breakdown Visualization

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#32004]
## User Story

_As a VFS product stakeholder, I would like to have a dashboard card that will show a categorized breakdown of all a11y failures in my application._

## Tasks
- [ ] Create a Domo Dashboard Card that reflects what categories the a11y failures with a given application fall into.
    * This data requires the prior `Evaluate Ingestion of GitHub A11y Issue Data Into BigQuery` task to be completed first. The data source for this should already exist in Domo.
    * This card should support filtering by application.
    * If possible, it'd be ideal to allow the filtering to be applied via clicking any of the bars to show a single application view, in addition to top-of-page filtering.

## Acceptance Criteria

- [ ] _A dashboard card exists reflecting what the categorical breakdown of a11y issues in a given application consists of._

**Comment by pjhill** [comment]
So pretty!
